images/singleuser/Dockerfile

```diff
@@ -75,6 +75,10 @@ RUN apt-get install --yes \
         lsof \
         mariadb-client
 
+# pyaudio
+RUN apt-get install --yes \
+        portaudio19-dev
+
```

Comment:
nit: at some point we might want to bundle all the apt install/update commands and clear the cache at the same time, to decrease the image file.

Reply:
Good thought!
https://phabricator.wikimedia.org/T302808

I'll leave the grouping alone for now. Though I agree there could be improvement there. For now I couldn't find a "And other stuff" section so I added another.